src/makefile

```diff
@@ -94,8 +94,11 @@ LOEK += example/script.cpp
 LOEK += test/audio.cpp
 LOEK += test/dummy.cpp
 JARO += test/PhysicsTest.cpp
-
-FMT := $(JARO) #<<< CHANGE THIS TO YOUR NAME FOR STEP 2
+WOUTER += crepe/api/LoopManager.h
```

Comment:
Nieuwe bestanden hoeven hier niet toegevoegd te worden omdat alle nieuwe bestanden door PR's moeten, en daardoor al aan de code standaard moeten voldoen. Je mag dit wel tijdelijk aanpassen om `make format` over de nieuwe code te draaien, maar zorg dat dit weer hersteld is bij de volgende aanpassingen.

Reply:
get_ticks is een private dus om hem te gebruiken moet ik een friend relatsie hebben. Ik ga wel even aan niels vragen waarom deze private zijn

Reply:
Volgensmij was dit een reactie op [dit](https://github.com/lonkaars/crepe/pull/28/files#r1836791288) commentaar, maar ik heb dan graag dat je dus in de code boven de `friend ...` even `//! LoopTimer uses \c get_ticks` oid neerzet zodat het altijd duidelijk blijft